frontend/src/app/[locale]/search/error.tsx

```diff
@@ -32,51 +41,34 @@ function isValidJSON(str: string) {
   }
 }
 
-function createBlankParsedError(): ParsedError {
-  return {
-    type: "NetworkError",
-    searchInputs: {
-      query: "",
-      status: "",
-      fundingInstrument: "",
-      eligibility: "",
-      agency: "",
-      category: "",
-      sortby: undefined,
-      page: "1",
-      actionType: "initialLoad",
-    },
-    message: "Invalid error message JSON returned",
-    status: -1,
-  };
-}
-
+// note that the SearchFilters component is not used here since that is a server component
+// we work around that by including the rendered components from SearchFilters, but manually
+// passing through the agency options as received from global state rather than fetching from API
```

Comment:
after discussing with Matt I realized that we could work around this by not using the Next's `error.js` pattern but use a normal React ErrorBoundary in a more targeted way. I think this is fine for now, but something to consider if this gets too hard to manage

Reply:
actually this kind of approach wouldn't really work since error boundaries require class components which don't work in Next server components 🤷